### ISSUE TITLE
sf_as_ee() error

### ISSUE THREAD

**Issue by giacfalk** [csaybar/rgee#52]
I am trying to read a Points shapefile ("saver.shp") as a FeatureCollection with sf_as_ee()  using the following lines of code:

```
ee_x <- read_sf("saver.shp") %>%
  st_transform(4326) %>%
  sf_as_ee()
```

Yet, I get the following error message:

```
Error in py_call_impl(callable, dots$args, dots$keywords) : 
  EEException: Unrecognized arguments {'geodesic', 'evenOdd'} to function: Constructs a new Point from the given x,y coordinates.

Args:
  coordinates: The coordinates of this Point in x,y order.
  crs: The coordinate reference system of the coordinates. The
      default is the projection of the inputs, where Numbers are
      assumed to be EPSG:4326. 
```

Any idea of what could be going on?

**Comment by csaybar** [comment]
Hi  @giacfalk, Thanks a lot for reporting us!. Are you using the rgee 0.3.10 version? Could I have access to the "saver.shp"?

**Comment by csaybar** [comment]
@giacfalk It was fixed in rgee 0.3.11. Please let me known if the problem persists.
